src/System.Diagnostics.TraceSource/src/project.json

```diff
@@ -15,7 +15,8 @@
         "System.Runtime.Extensions": "4.4.0-beta-24714-01",
         "System.Runtime.InteropServices": "4.4.0-beta-24714-01",
         "System.Threading": "4.4.0-beta-24714-01",
-        "System.Threading.Tasks": "4.4.0-beta-24714-01"
+        "System.Threading.Tasks": "4.4.0-beta-24714-01",
+        "System.Security.Permissions": "4.4.0-beta-24714-01"
```

Comment:
We should remove System.Security.Permissions dependency. If there are CAS attributes applied they should be removed to break this dependency.